### ISSUE TITLE
Consider removing/changing aws_s3_bucket_invalid_region check

### ISSUE THREAD

**Issue by FnTm** [wata727/tflint#441]
I just tried to do the first linting on my project, and got an error:
```
Error: region is not a valid value (aws_s3_bucket_invalid_region)
  on x.tf line 202:
 202:   region   = "us-east-2"
```
But the bucket is definitely there, and this error seems to be a false-positive.

Digging into this a bit I found that:
- This seems to be coming from https://github.com/wata727/tflint/blob/283037d25804adfd3032e3857121d8e0cd6e98d3/rules/awsrules/models/aws_s3_bucket_invalid_region.go
- Which in turn generates that list from a constant named `BucketLocationConstraint`
- Which is mentioned in the aws-sdk-cpp as outdated in 2015: https://github.com/aws/aws-sdk-cpp/issues/48
- In addition I can't seem to find any AWS region that doesn't have S3 at the moment: https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/

Therefore I'd propose either dropping this check altogether, or updating it to use some other constant to determine whether a region is valid.

**Comment by wata727** [comment]
Good catch, @FnTm. I opened a pull request to update the list of regions. https://github.com/wata727/tflint/pull/442

**Comment by FnTm** [comment]
@wata727 thanks for the super quick response! Eagerly waiting for the PR to be merged to I can add tflint to our CI without this exception :)

**Comment by wata727** [comment]
v0.11.2 has been released. Please try this.
FYI, if you would like to ignore these false positives, you can use `tflint-ignore` annotation comment. See https://github.com/wata727/tflint/blob/v0.11.2/docs/guides/config.md#rule-blocks

**Comment by FnTm** [comment]
Wonderful, thank you for both the quick fix (I've confirmed that it does indeed fix the issue) and pointing me to the annotation.
